### PR TITLE
[desktop] Changelog for 2022.05.31-10

### PR DESCRIPTION
**Pull request by fgaz**

### CODE REVIEW

File: qt/res/app.organicmaps.desktop.metainfo.xml [hunk 1]
```diff
@@ -107,6 +107,14 @@
   </screenshots>
 
   <releases>
+   <release version="2022.05.31-10" date="2022-05-31">
+     <description>
+       <ul>
+         <li>Do not draw place=region.</li>
+         <li>Fix crash on routing.</li>
```
Comment (fgaz) [inline qt/res/app.organicmaps.desktop.metainfo.xml:114]: I took the liberty of adding this since it's a pretty important fix for flatpak

Comment (biodranik) [reply qt/res/app.organicmaps.desktop.metainfo.xml:114]: Did we fix the crash?

Comment (fgaz) [reply qt/res/app.organicmaps.desktop.metainfo.xml:114]: According to https://github.com/flathub/app.organicmaps.desktop/issues/2 the crash was fixed in https://github.com/organicmaps/organicmaps/pull/2625, which is included in 2022.05.31-10: https://github.com/organicmaps/organicmaps/commit/1a631163449fbecd0182c5f4a57123b63bf48bda

image

Comment (biodranik) [reply qt/res/app.organicmaps.desktop.metainfo.xml:114]: Good, sorry I forgot about it :)

**Review by biodranik — APPROVED** [review]

**Pull request merged**